### PR TITLE
Add example for virtualenv integration in cli docs

### PR DESCRIPTION
**Pull request by greyli**

### CODE REVIEW

**Comment by untitaker** [comment]
Editing `activate` scripts is not a good idea, it is like editing code in `site-packages`.

**Comment by greyli** [comment]
Thanks. This part [Virtualenv Integration](http://flask.pocoo.org/docs/0.11/cli/#virtualenv-integration) was already in docs, is it better to remove it?

**Comment by untitaker** [comment]
I see, I wasn't aware of that section.

File: docs/cli.rst [hunk 1]
```diff
@@ -56,6 +56,18 @@ If you are constantly working with a virtualenv you can also put the
 bottom of the file.  That way every time you activate your virtualenv you
 automatically also activate the correct application name.
 
+You need to edit the specific activate script for different shell, for example:
```
Comment (untitaker) [inline docs/cli.rst]: This is not related to the shell.

Comment (davidism) [inline docs/cli.rst]: There are different scripts depending on the shell you're using. This sentence is awkward though, should read "Edit the activate script for the shell you use. For example:"

Comment (greyli) [inline docs/cli.rst]: There are different activate script in there (e.g. activate, activate.csh, activate.fish...) for different shell.

Comment (greyli) [inline docs/cli.rst]: Edited.

Comment (untitaker) [inline docs/cli.rst]: Right, I was under the impression that `activate` was POSIX shell compatible.

File: docs/cli.rst [hunk 2]
```diff
@@ -56,6 +56,18 @@ If you are constantly working with a virtualenv you can also put the
 bottom of the file.  That way every time you activate your virtualenv you
 automatically also activate the correct application name.
 
+Edit the activate script for the shell you use, for example:
+
+Unix Bash: ``venv/bin/activate``::
+
+    FLASK_APP=foo.py
```
Comment (davidism) [inline docs/cli.rst]: I'd rather the examples encourage the use of installed package names rather than file names. `foo` instead of `foo.py`. @untitaker thoughts?

Comment (untitaker) [inline docs/cli.rst]: Agreed. Perhaps replace with `...` or `<see above>`?

Comment (davidism) [inline docs/cli.rst]: Or `hello`, since that's what's used in the other examples.

Comment (greyli) [inline docs/cli.rst]: How about `<your app>`?

Comment (untitaker) [inline docs/cli.rst]: Yes, all good

File: docs/cli.rst [hunk 3]
```diff
@@ -56,6 +56,18 @@ If you are constantly working with a virtualenv you can also put the
 bottom of the file.  That way every time you activate your virtualenv you
 automatically also activate the correct application name.
 
+Edit the activate script for the shell you use. For example:
+
+Unix Bash: ``venv/bin/activate``::
+
+    FLASK_APP=hello
+    export FLASK_APP
```
Comment (davidism) [inline docs/cli.rst]: Should just be `export FLASK_APP=hello`, no need to split the lines in this case (and most cases).

Comment (greyli) [inline docs/cli.rst]: The generated code above was in this form, is it better to keep it?

Comment (davidism) [inline docs/cli.rst]: Yeah, it's fine either way.

**Comment by untitaker** [comment]
Thanks @greyli 🎉
